src/content/guides/workload-cluster-labelling/index.md

```diff
@@ -12,6 +12,10 @@ owner:
 
 # Labelling workload clusters
 
+{{< platform_support_table aws="ga=v10.0.0" azure="ga=v13.0.0" >}}
```

Comment:
@giantswarm/team-rocket I am assuming that KVM will get this feature once there is a Cluster API based resource for the workload cluster on KVM. Is there a roadmap issue for this we can link to? I didn't find one.

Reply:
This is our umbrella CAPI issue https://github.com/giantswarm/giantswarm/issues/13162

Reply:
Thanks! For this purpose here we can only use public issues in giantswarm/roadmap.